packages/wonder-blocks-tabs/package.json

```diff
@@ -16,14 +16,14 @@
     "access": "public"
   },
   "dependencies": {
-    "@babel/runtime": "^7.24.5",
+    "@babel/runtime": "catalog:",
```

Comment:
Updating to use catalogs (these changes were in a feature branch when https://github.com/Khan/wonder-blocks/pull/2505 was implemented!)